autoload/codi.vim

```diff
@@ -693,3 +694,9 @@ function! codi#run(bang, ...)
     return s:codi_spawn(filetype)
   endif
 endfunction
+
+function! codi#log(message)
```

Comment:
I would prefer to move this function to the top of the file, to group it with `s:err`. I also would prefer to keep the function script-local, since I feel like it doesn't make sense to manually invoke it.